app/views/admin/tutorials/new.html.erb

```diff
@@ -15,4 +16,13 @@
   <% end %>
 
   <%= f.submit "Save", class: "mt2 btn btn-primary mb1 bg-teal"  %>
+
+  <h1>Add New Youtube Playlist</h1>
+  <section class="import-youtube">
+    <%= f.fields_for :playlist do |builder| %>
+      <%= f.label 'Playlist ID' %>
+      <%= f.text_field :playlist_id, class: "block col-4 field" %>
+    <% end %>  
+    <%= link_to 'Import Youtube Playlist', "#", "data-action" => "click->tutorials#updateOrder", class: "video-position-submit btn btn-outline", id: @tutorial.id %> 
```

Comment:
Looked through other files and copied that set up for this code here. This also ensures the css is the same for the new pages we make